src/duckling/game/collision/collision-attribute.ts

```diff
@@ -1,26 +1,51 @@
 import {Attribute, Entity} from '../../entitysystem/entity';
 import {Vector} from '../../math/vector';
 import {Box2} from '../../math/box2';
+import {SelectOption} from '../../controls';
 
 export const COLLISION_KEY = "collision";
 
 /**
  * Describes how the entity behaves in the solid body system.
  */
-export enum BodyType {
-    None,
-    Solid,
-    Environment
-}
+export type BodyType = "none" | "solid" | "environment";
```

Comment:
Make a toSelectOption(["none", "solid", "environment"]), use it in addition to the BodyType type define.